### PR TITLE
chore: added new resource in illustrations

### LINKED ISSUE #664: [Add] Free SVG illustrations
### Site URL

https://freesvgillustration.com/

### Description of the above provided site

Free SVG illustrations are available for any idea you can imagine. You can change the primary and secondary colours of the illustrations to match your project's style guide. Download the illustrations in either PNG or SVG format. premium illustrations are also available.
Here's an example of illustration.
image

### Category

Frontend

### Sub Category

"illustrations"

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Nilesh9106**
## Fixes Issue

Closes #664 

## Changes proposed
- added new resource in illustrations

## Screenshots
image

### CODE REVIEW

File: database/frontend/illustrations.json [hunk 1]
```diff
@@ -82,5 +82,12 @@
     "url": "https://opendoodles.com/",
     "category": "frontend",
     "subcategory": "illustrations"
+  },
+  {
+    "name": "Free SVG illustrations",
+    "description": "Free SVG illustrations are available for any idea you can imagine. You can change the primary and secondary colors of the illustrations to match your project's style guide. Download the illustrations in either PNG or SVG format. premium illustrations are also available.",
```
Comment (CBID2) [inline database/frontend/illustrations.json:88]: [Typo]
```suggestion
    "description": "Free SVG illustrations are available for any idea you can imagine. You can change the primary and secondary colors of the illustrations to match your project's style guide. Download the illustrations in either PNG or SVG format. Premount illustrations are also available.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Nilesh9106! :) Just fix the typo and you’ll be ready to merge! :)

**Comment by Nilesh9106** [comment]
hii @CBID2. I have fixed the requested changes. can you please review PR?

**Review by rupali-codes — APPROVED** [review]
looks good

**Pull request merged**